src/ctia/entity/feed.clj

```diff
@@ -391,7 +391,7 @@
             :identity-map identity-map
             (ok (-> (get-store :feed)
                     (query-string-count
-                     (routes.common/search-query :created params)
+                     (routes.common/search-query {:date-field :created, :params params})
```

Comment:
```suggestion
                     (routes.common/search-query {:date-field :created :params params})
```
Just occurred to me, though, are the commas intentional because they're on the same line?

Reply:
Yes, I'll change them to newlines.